nifi-extension-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/drive/ListGoogleDrive.java

```diff
@@ -197,7 +266,37 @@ public void onScheduled(final ProcessContext context) throws IOException {
 
         HttpTransport httpTransport = new ProxyAwareTransportFactory(proxyConfiguration).create();
 
-        driveService = createDriveService(context, httpTransport, DriveScopes.DRIVE_METADATA_READONLY);
+        driveService = createDriveService(context, httpTransport, DriveScopes.DRIVE, DriveScopes.DRIVE_METADATA_READONLY);
+    }
+
+    @OnScheduled
+    @Override
+    // initListedEntityTracker() needs to be overridden because the super implementation calls getRecordSchema()
+    // so recordSchema must be initialized before that
+    public void initListedEntityTracker(ProcessContext context) {
+        this.enrichAttributes = new ArrayList<>();
+
+        final List<RecordField> recordFields = new ArrayList<>();
+
+        Stream.of(
+                GoogleDriveFlowFileAttribute.ID,
+                GoogleDriveFlowFileAttribute.FILENAME,
+                GoogleDriveFlowFileAttribute.SIZE,
+                GoogleDriveFlowFileAttribute.SIZE_AVAILABLE,
+                GoogleDriveFlowFileAttribute.TIMESTAMP,
+                GoogleDriveFlowFileAttribute.CREATED_TIME,
+                GoogleDriveFlowFileAttribute.MODIFIED_TIME,
+                GoogleDriveFlowFileAttribute.MIME_TYPE
+        ).forEach(a -> recordFields.add(a.getRecordField()));
```

Comment:
Creating a new `Stream` seems unnecessary, is it possible to simply iterative over these values, or if required, creating a static `List.of()`?